students/BrianB/lesson02/series.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+
+#--------------------------------------------------
+# Who: Brian Brumleve
+# What: Lesson02 - Series
+# When: April 17, 2019
+# Program Description:
+#   This program uses a common script to develop
+#   a listing of Fibonacci Series numbers and
+#   Lucas Numbers.  The program is also capable
+#   of creating a unique series of numbers.
+#--------------------------------------------------
+
+
+# Fibonacci Series
+#   This function assigns optional parameters (OP)
+#   specific to the Fibonacci Series.  These
+#   optional parameters represent the first two
+#   values of F(0) and F(1).  The math for
+#   developing the Fibonacci Series is performed
+#   in the sum_series() function, which is called
+#   to by the fibonacci() function.
+def fibonacci(n, a=0, b=1):
+    return sum_series(n, a, b)
```

Comment:
yes!

except the Fibonacci series always starts with (0, 1), so this should not take optional a and b parameters.

This is essentially exactly the same as the series function ...